react/Button/ButtonGroup.demo.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
This seemed a bit odd to me, seeming as `group` is just a prop on Button it felt like it should be a component of the Button demo. Can appreciate though that this might be cleaner to do.